### ISSUE TITLE
WIS to AC not working on respec'd Sosiel

### ISSUE THREAD

**Issue by serlex00** [BarleyFlour/RespecMod#73]
Hello: I have respec'ed Sosiel to Ecllesithurge + Zen Archer, and It is supposed to get  WIS to AC bonus on level one while unecumbred and unarmored and it is not working

![Pathfinder  Wrath of the Righteous Screenshot 2021 09 16 - 13 31 50 22](https://user-images.githubusercontent.com/35397401/133605333-1a2d8736-5d35-4ff7-aaca-50b93e44adcc.png)
![Pathfinder  Wrath of the Righteous Screenshot 2021 09 16 - 13 31 54 42](https://user-images.githubusercontent.com/35397401/133605337-be94fbba-26c4-440e-a585-2e04edfdf59c.png)

**Comment by Alosno23** [comment]
Doesn't it? I mean, it dont show in the listed modifiers to AC (and i get your confusion in this point) but it's there. You have 5 points of WIS. Your AC modifiers displays 10 + 3 (DEX), but you have 18 total.

**Comment by Alosno23** [comment]
Yeah. I tested it with Respec'd Woljif Sword Saint and it works, but just dont display it.
